src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -4316,6 +4316,19 @@ paths:
       parameters:
         - $ref: '#/parameters/personIdParam'
         - $ref: '#/parameters/fieldsParam'
+        - name: client
+          in: query
```

Comment:
I'm afraid this is not the correct approach, for POST APIs the body should be used to send values to the API, the query parameters are only supposed to be used for controlling the response i.e. paging, fields etc.

To send in extra data you should be updating the "siteMembershipRequestBodyCreate" body definition with new additional properties i.e. "client" and "workspacePath"